src/generator/templates/api-endpoint.njk

```diff
@@ -20,10 +20,12 @@ import {
   GoogleConfigurable,
   createAPIRequest,
   MethodOptions,
+  StreamMethodOptions,
   GlobalOptions,
   BodyResponseCallback,
   APIRequestContext,
 } from 'googleapis-common';
+import {Readable} from 'stream';
```

Comment:
[suggestion, feel free to ignore]
should this be imported only if used by the given API? (otherwise, unused imports)

I must admit I have no idea if this applies to APIs in this library, but in GAPICs, we do put all such imports inside if statements in templates.

Reply:
Yeah, in this particular case the import is *only* being used for the `Readable` type.  It could be required for any of the API calls, if the user uses `{responseType: 'stream'}`